test/support/utils_sequence.h

```diff
@@ -24,7 +24,7 @@
 
 #include "iterator_utils.h"
 
-// Please uncomment this define if required to print full contect of sequence.
+// Please uncomment this define if required to print full context of sequence.
```

Comment:
I think here it should be "content", not "context"

Reply:
Ok,  I believe content works better as well, but that means we should also change the `#define PRINT_FULL_SEQUENCE_CONTEXT 1`  -> `#define PRINT_FULL_SEQUENCE_CONTENT 1`.  

I'll make the change.